content/en/docs/concepts/storage/storage-classes.md

```diff
@@ -110,6 +111,29 @@ either `Delete` or `Retain`. If no `reclaimPolicy` is specified when a
 Persistent Volumes that are created manually and managed via a storage class will have
 whatever reclaim policy they were assigned at creation.
 
+### Allow Volume Expansion
+
+{{< feature-state for_k8s_version="v1.15" state="beta" >}}
+
+Persistent Volumes can be configured to be expandable. This feature when set to `true`, 
+allows the users to resize the volume by editing the corresponding PVC object. 
+
+The following types of volumes support volume expansion, when the underlying
+Storage Class has the field `allowVolumeExpansion` set to true.
+
+* gcePersistentDisk
+* awsElasticBlockStore
+* Cinder
+* glusterfs
+* rbd
+* Azure File
+* Azure Disk
+* Portworx
+* FlexVolumes
+* CSI {{< feature-state for_k8s_version="v1.15" state="alpha" >}}
```

Comment:
csi is alpha in 1.14

Reply:
Thank you @msau42 . I have corrected the version